vioscsi/helper.c

```diff
@@ -384,46 +393,48 @@ ENTER_FN();
                 SP_INTERNAL_ADAPTER_ERROR,
                 __LINE__);
         RhelDbgPrint(TRACE_LEVEL_FATAL, " CANNOT READ PCI CONFIGURATION SPACE %d\n", pci_cfg_len);
+EXIT_FN();
```

Comment:
The issue was that in the trace I could see entry into the functions, but when leaving early, it indicated a potential problem, because the corresponding exit trace was missing.

Reply:
Jon, if you were actually talking about the section below, it was because it was in orphaned braces.